src/Form/TripType.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Form;
+
+use App\Entity\Trip;
+use App\Entity\User;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class TripType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('date')
+            ->add('addressStart')
+            ->add('addressEnd')
+            ->add('nbPassengers')
+            ->add('driver', EntityType::class, [
+                'class' => User::class,
+                'choice_label' => 'id',
+                'multiple' => false,
+                'expanded' => false,
+                'by_reference' => false,
+            ])
```

Comment:
this shouldn't be an option in the form: the driver will be the connected user. since you don't have authentication in the project, you should just remove this field from the entity and add it later